src/scheduler/limits.cpp

```diff
@@ -1158,13 +1115,13 @@ void update_soft_limits(server_info *si, queue_info *qi, resource_resv *rr)
  * @return	int
  * @retval	Accumulated preempt_bits matching the entity
  */
-int find_preempt_bits(counts *entity_counts, const char *entity_name, resource_resv *rr)
+int find_preempt_bits(counts_map &entity_counts, const char *entity_name, resource_resv *rr)
 {
 	counts *cnt = NULL;
 	resource_count *res_c;
 	int rc = 0;
 
-	if (entity_counts == NULL || entity_name == NULL)
+	if (entity_counts.size() == 0 || entity_name == empty_str)
```

Comment:
entity_name is a char *.  I don't think you meant this.  You'd check it against NULL.  In any case, you don't need this overload.

Reply:
oops :)